### DISCUSSION TITLE
Reminder: Gazebo usage

### DISCUSSION THREAD

**Discussion by chaidop** [gstavrinos/ai_msc_robotic_systems_assignments#50] [Q&A] [answered]
Καλησπέρα,
είχε ειπωθεί στο παρελθόν, αλλά μπορεί κάποιος να μου υπενθιμισει πως ανοιγω το gazebo simulation για το assignment 3?

**Comment by gstavrinos** [comment, marked as answer]
Στο τέλος της εντολής που τρέχεις, προσθέτεις `gui:=true`

**Reply by chaidop** [reply]
Thank you 🙏🙏
